### ISSUE TITLE
Limitation of number of characters for (?<dimension>xxx) group in HTML grabber

### ISSUE THREAD

**Issue by chfnd** [Rello/analytics#96]
Is there a limitation in the number of characters for the `(?<dimension>xxx)` group when using **HTML grabber** for dataload?
I noticed I cannot load data if the group has more than _64_ characters.

**Example 1: works**
URL: https://www.santepubliquefrance.fr/dossiers/coronavirus-covid-19/coronavirus-chiffres-cles-et-evolution-de-la-covid-19-en-france-et-dans-le-monde
valid regex: `/(<\/p><h3>)(?<dimension>.{64})(.*?)(<\/h3>)(.*?)(<strong>)(?<value>[\d\s]*)(&nbsp;<\/strong>)/`

**Example 2: does NOT work**
URL: https://www.santepubliquefrance.fr/dossiers/coronavirus-covid-19/coronavirus-chiffres-cles-et-evolution-de-la-covid-19-en-france-et-dans-le-monde
valid regex: `/(<\/p><h3>)(?<dimension>.{65})(.*?)(<\/h3>)(.*?)(<strong>)(?<value>[\d\s]*)(&nbsp;<\/strong>)/`

Notice the difference of number of characters in the dimension group.

**Comment by Rello** [comment]
Hello,
this is correct, because the dimensions only have 64 characters because it would be difficult to create a meaningful display in the charts or tables

If I am not wrong, you want to extract:
`Evolution du coronavirus (Sars-CoV-2) Monde et Europe : les chiffres clés en semaine 52 (source : ECDC)`

what are you trying to display in the report then?

**Comment by chfnd** [comment]
> Hello,
> this is correct, because the dimensions only have 64 characters because it would be difficult to create a meaningful display in the charts or tables

Makes sense. I think it would be nice to put the info in the wiki to avoid similar questions.

> If I am not wrong, you want to extract:
> `Evolution du coronavirus (Sars-CoV-2) Monde et Europe : les chiffres clés en semaine 52 (source : ECDC)`
> 
> what are you trying to display in the report then?

No, it is just an example to illustrate my point.
The data I was looking to extract is `nombre de personnes positives déclarées en 24h à J-3 (données consolidées)` (for a daily report) and I managed to do it with this regex:
`/(?<dimension>nombre de personnes positives d&eacute;clar&eacute;es en 24h)(.*?)(<strong>)(?<value>.*?)(<\/strong>)/`

I had to trim the dimension though, mainly because of the accented characters.
One thing that could help is the use of HTML legends for charts. As for this issue, I am closing it. Thanks!
